update-selectors.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+node ./zenbot.sh backfill bitfinex.ETH-USD --days 365
+node ./zenbot.sh backfill bitfinex.BTC-USD --days 365
+node ./zenbot.sh backfill bitfinex.ETH-BTC --days 365
+node ./zenbot.sh backfill bitfinex.LTC-USD --days 365
+node ./zenbot.sh backfill bitfinex.XRP-USD --days 365
+
+node ./zenbot.sh backfill kraken.XETH-ZCAD --days 365
+node ./zenbot.sh backfill kraken.XETH-ZUSD --days 365
+node ./zenbot.sh backfill kraken.XXBT-ZCAD --days 365
+node ./zenbot.sh backfill kraken.XXBT-ZCAD --days 365
+node ./zenbot.sh backfill kraken.XETH-XXBT --days 365
```

Comment:
if you do not wish to include this file in your PR, you can just `git rm update-selectors.sh` and the usual `git add --all` `git commit` `git push`